k8s/perftest/common/probate/probate-back-office.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-back-office
+  namespace: probate
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.java: glob:prod-*
+spec:
+  releaseName: probate-back-office
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    name: probate-back-office
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    version: 1.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: "hmctspublic.azurecr.io/probate/back-office:prod-e9c0c00b"
+      environment:
+        CCD_GATEWAY_HOST: https://manage-case.perftest.platform.hmcts.net
```

Comment:
think it really should be the gateway on for now - until we make the XUI switch

CCD_GATEWAY_HOST: https://gateway-ccd.perftest.platform.hmcts.net